cpp/include/cudf/io/json.hpp

```diff
@@ -203,6 +206,13 @@ class json_reader_options {
    */
   bool is_enabled_experimental() const { return _experimental; }
 
+  /**
+   * @brief Whether the experimental reader should keep quotes of string values.
+   *
+   * @returns true if the experimental reader should keep quotes, false otherwise
+   */
+  bool is_keeping_quotes() const { return _keep_quotes; }
```

Comment:
Consistent with other options.
```suggestion
  bool is_enabled_keep_quotes() const { return _keep_quotes; }
```

Reply:
Fixed 👍